### ISSUE TITLE
Wii port - Large NeoGeo games support

### ISSUE THREAD

**Issue by SieKensou** [libretro/fbalpha2012_neogeo#22]
Would be possible to add that stuff to this stand alone versions for the Nintendo Wii? It would be great to have this implemented: https://gbatemp.net/threads/large-neogeo-games-support-and-cps3-tests.493767/

P.S.: Could be this FBA stand alone versions of NeoGeo, CPS-1 and CPS-2 be already compiled for the Wii in a downloads section?

Thanks for all this great work!

**Comment by andres-asm** [comment]
I don't understand the request.
Are you asking for FBA standalone?

**Comment by SieKensou** [comment]
Yes, it would be great to have stand alone versions of FBA for CPS-2 and NeoGeo. I recently found a stand alone version of FBA CPS-1 and works really well but I don't know how to compile them (that's why I asked for the compiled versions for the Wii in a download sections).
About the first question is because big games doesn't load in the Wii version because its memory, but in the link I put, the member Wiimpathy added a way to load this big games using a cache system.

**Comment by SieKensou** [comment]
Something new about this? Who is the person that is working in the updates of FBA NeoGeo core for the Wii?
